src/sentry_plugins/trello/plugin.py

```diff
@@ -0,0 +1,260 @@
+(at your option) any later version.
+
+Sentry-Trello is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Sentry-Trello. If not, see <http://www.gnu.org/licenses/>.
+"""
+from __future__ import absolute_import
+
+# import sentry_trello
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from rest_framework.response import Response
+from requests.exceptions import RequestException
+
+from sentry.exceptions import PluginError, PluginIdentityRequired
+from sentry.plugins.base import JSONResponse
+from sentry.plugins.bases.issue import IssuePlugin, NewIssueForm
+from sentry.exceptions import PluginError
+from sentry.utils.http import absolute_uri
+
+from sentry.plugins.bases.issue2 import IssuePlugin2, IssueGroupActionEndpoint
+from sentry_plugins.base import CorePluginMixin
+from .client import TrelloClient
+
+SETUP_URL = 'https://github.com/getsentry/sentry-trello/blob/master/HOW_TO_SETUP.md'  # NOQA
+
+ISSUES_URL = 'https://github.com/getsentry/sentry-trello/issues'
+
+EMPTY = (('', '--'),)
+
+ERR_AUTH_NOT_CONFIGURED = (
+    'You still need to associate a Trello identity with this account.'
+)
+
+
+class TrelloError(Exception):
+    status_code = None
+
+    def __init__(self, response_text, status_code=None):
+        if status_code is not None:
+            self.status_code = status_code
+        self.text = response_text
+        super(TrelloError, self).__init__(response_text[:128])
+
+    @classmethod
+    def from_response(cls, response):
+        return cls(response.text, response.status_code)
+
+
+class TrelloPlugin(CorePluginMixin, IssuePlugin2):
+    description = 'Create Trello cards on issues'
+    slug = 'trello'
+    title = 'Trello'
+    conf_title = title
+    conf_key = 'trello'
+    auth_provider = None
+    allowed_actions = ('create',)
+
+    def get_config(self, project, **kwargs):
+        def get_from_initial(initial, field):
+            return initial.get(field) or self.get_option(field, project)
+
+        initial = kwargs.get('initial') or {}
+        key_value = get_from_initial(initial, 'key')
+
+        key = {
+            'name': 'key',
+            'label': 'Trello API Key',
+            'type': 'text',
+            'required': True,
+            'default': key_value,
+        }
+        token = {
+            'name': 'token',
+            'label': 'Trello API Token',
+            'type': 'secret',
+            'required': True,
+        }
+        token_value = get_from_initial(initial, 'token')
+
+        if token_value:
+            token['has_saved_value'] = True
+            token['prefix'] = token_value[:6]
+            token['required'] = False
+
+        config = [key, token]
+
+        if key_value and token_value:
```

Comment:
@billyvg I have some fears this will not work. The idea way back when was to create a "dynamic" Django form. So, if `[key, token]` are present try to get the organizations. Wrote a lot of this to work around the fact that the move away from Django forms broke Trello. I don't know what is on your plate right now, or if it's even worth making a more permanent fix here, but this really needed a react solution, and I was making due with the skills I had.